src/utils.jl

```diff
@@ -1,18 +1,33 @@
 """
-    highword(d::Float64)
+    highword(x::Union{Float32,Float64}) -> UInt32
 
-Get the most significant 32 bits as a `UInt32` from `d`.
+Get the most significant 32 bits as a `UInt32` from `x`.
 Corresponds to `GET_HIGH_WORD` in musl
 """
-highword(d::Float64) = (reinterpret(UInt64, d) >> 32) % UInt32
+@inline highword(x::Float64) = (reinterpret(UInt64, x) >> 32) % UInt32
```

Comment:
tbh  the % UInt32 is actually not needed (code_native)  it's probably far more readable to just write  UInt32(reinterpret(UInt64, x) >> 32)   I got rid of these % conversions in sleef when I realized they didnt' have any speed benefits..